app/views/proposals/show.html.haml

```diff
@@ -97,3 +97,11 @@
               %dt Requires Registration:
               %dd
                 = link_to "Yes (#{registered_text(@event)})", new_conference_conference_registration_path(@conference.short_title), class: 'btn btn-xs btn-danger', disabled: !@event.registration_possible?
+          - if concurrent_events(@event).present?
+            .col-md-12#concurrent-events
+              %h4
+                = "Other Events on \n #{@event_schedule.start_time.strftime("%Y %B %e")} at #{@event_schedule.start_time.strftime("%H:%M")}:"
```

Comment:
Events is a term we only use in the admin namespace of OSEM, its meaning is not very straightforward to attendees.
This could simply be
**Happening at the same time**

As we are mentioning the date and time a few lines above.